windows-nsis/openvpn.nsi

```diff
@@ -244,7 +250,11 @@ Section /o "${PACKAGE_NAME} Service" SecService
 
 	SetOutPath "$INSTDIR\bin"
 	File /oname=openvpnserv2.exe "${OPENVPNSERV2_EXECUTABLE}"
-	File "${OPENVPN_ROOT}\bin\openvpnserv.exe"
+        ${If} ${RunningX64}
+                File "${OPENVPN_ROOT_X86_64}\bin\openvpnserv.exe"
+        ${Else}
+                File "${OPENVPN_ROOT_I686}\bin\openvpnserv.exe"
+        ${EndIf}
```

Comment:
git hooks are not platform independent. it is hard to make it work on unix/osx/windows
but we can do that with either Travis-CI or https://reviewable.io

I'll push a patch (with tabs) soon